app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+
+  def destroy
+    @review = Review.find(params[:id])
+    @wizard = @review.wizard
+    if (current_user.id == @review.user_id) || current_user.role == 'admin'
```

Comment:
good! You could consider creating a `destroyable_by?(user)` instance method in your model that returns true or false based on these conditions. That way the behavior can be unit tested